doc/jsk_perception/nodes/image_publisher.md

```diff
@@ -21,6 +21,13 @@ Publish image from loaded file.
 * `~frame_id` (str default: `camera`)
 * `~rate` (Float default: `1.0`)
 
+* `~fovx` (Float default: `None`)
+* `~fovy` (Float default: `None`)
+
+  If `publish_info` is `True` and `fovx` and `fovy` are specified,
```

Comment:
`publish_info` -> `~publish_info`
`fovx` -> `~fovx`
`fovy` -> `~fovy`
And could you please update line 16?

Reply:
Modified